scripts/ethers/withdrawNativeAnchor.js

```diff
@@ -30,14 +30,15 @@ const recipientAddress = process.argv[4];
 const wallet = new ethers.Wallet(process.env.PRIVATE_KEY, provider);
 const anchorAbi = require("../../build/contracts/Anchor.json");
 const anchorInstance = new ethers.Contract(contractAddress, anchorAbi.abi, wallet);
+const MERKLE_TREE_HEIGHT = 20;
 
 async function generateMerkleProof(deposit) {
   const events = await getDepositEvents(contractAddress);
 
   const leaves = events
     .sort((a, b) => a.args.leafIndex - b.args.leafIndex) // Sort events in chronological order
     .map(e => e.args.commitment);
-  const tree = new MerkleTree(process.env.MERKLE_TREE_HEIGHT, leaves);
+  const tree = new MerkleTree(MERKLE_TREE_HEIGHT, leaves);
```

Comment:
Are all deployments using height of 20? We should also add this to the docs if so.